core/modules/widgets/checkbox.js

```diff
@@ -215,6 +215,8 @@ CheckboxWidget.prototype.handleChangeEvent = function(event) {
 		if($tw.utils.isArray(fieldContents)) {
 			// Make a copy so we can modify it without changing original that's refrenced elsewhere
 			listContents = fieldContents.slice(0);
+		} else if(fieldContents == undefined) {
```

Comment:
we usually use `fieldContents === undefined` to avoid type casting.

Reply:
I wondered if `null` should also be considered as equivalent to an empty list. But I suppose if someone has gone to great effort to put a `null` into a TW field (which would be very hard unless you're trying to do it on purpose), they can just deal with the fact that it doesn't work with the checkbox widget. I'll change this to `===`.